base/asyncevent.jl

```diff
@@ -9,7 +9,8 @@ Create a async condition that wakes up tasks waiting for it
 (by calling [`wait`](@ref) on the object)
 when notified from C by a call to `uv_async_send`.
 Waiting tasks are woken with an error when the object is closed (by [`close`](@ref)).
-Use [`isopen`](@ref) to check whether it is still active.
+Use [`isopen`](@ref) to check whether it is still active. A closed condition will
```

Comment:
Maybe this would be better? Yes, the function is called `isopen` but the text says "active", not "open?
```suggestion
Use [`isopen`](@ref) to check whether it is still active. An inactive condition will
```

Reply:
OK I see elsewhere the text already talks about "closed". I find this use of two different terms meaning the same thing adds a layer of potential confusion. So perhaps like this:
```suggestion
Use [`isopen`](@ref) to check whether it is still active. A closed condition is inactive and will
```